src/features/klant/bedrijf/service/use-bedrijf-by-identifier.ts

```diff
@@ -60,6 +46,10 @@ const getUrlVoorGetBedrijfById = (
     "nietNatuurlijkPersoonIdentifier" in bedrijfsZoekParamter &&
     bedrijfsZoekParamter.nietNatuurlijkPersoonIdentifier
   ) {
+    const identifier =
+      bedrijfsZoekParamter.nietNatuurlijkPersoonIdentifier.length === 9
```

Comment:
als we de klant ophalen, dan weten we waar die vandaan kwam en dus of dit een rsin of kvknr is. Als we dat bij de klant bijhouden en aan useBedrijfByIdentifier een functie meegeven die niet alleen de waarde van de identifier, maar ook de soort retourneert, dan hoeven we hier niet obv de lengte te raden wat voor soort het is.

Reply:
Dit was voor nu het snelste, als we de OK2 bedrijf stories uitvoeren dan hoeven we op de frontend nergens meer een nietNatuurlijkPersoonIdentifier te gebruiken. Dan wil ik dit stukje ook opruimen.

Reply:
Besproken, gaan we een comment bijzetten